LeaderboardBackend.Test/Categories.cs

```diff
@@ -288,4 +289,135 @@ public static async Task CreateCategory_BadData(string? name, string? slug, obje
         problemDetails.Should().NotBeNull();
         problemDetails!.Title.Should().Be("One or more validation errors occurred.");
     }
+
+    [Test]
+    public static async Task DeleteCategory_OK()
+    {
+        IServiceScope scope = _factory.Services.CreateScope();
+        ApplicationContext context = scope.ServiceProvider.GetRequiredService<ApplicationContext>();
+
+        Category cat = new()
+        {
+            Name = "Delete Cat OK",
+            Slug = "deletecat-ok",
+            LeaderboardId = _createdLeaderboard.Id,
+            SortDirection = SortDirection.Ascending,
+            Type = RunType.Time
+        };
+
+        context.Add(cat);
+        await context.SaveChangesAsync();
+        cat.Id.Should().NotBe(default);
+        context.ChangeTracker.Clear();
```

Comment:
This is correct, right? I want to clear the change tracker as otherwise the check at the bottom of the test will fail